ui/src/utils/connectrpc-extension.ts

```diff
@@ -0,0 +1,26 @@
+import { Timestamp, timestampDate } from '@bufbuild/protobuf/wkt';
+
+import { Time } from '@ui/gen/k8s.io/apimachinery/pkg/apis/meta/v1/generated_pb';
+
+export const k8sApiMachineryTimestampDate = (t?: Time | PlainMessage<Time> | string) => {
```

Comment:
Can we find a better name for it? Do we use it for all dates or just selected ones?